src/pymoc/modules/column.py

```diff
@@ -231,7 +231,7 @@ def vertadvdiff(self, wA, dt, do_conv=False):
     dz = self.z[1:] - self.z[:-1]
 
     # apply boundary conditions:
-    if do_conv is None and not self.do_conv or not do_conv:
+    if (do_conv is None and not self.do_conv) or not do_conv:
```

Comment:
I still don't think this does what we want. If do_conv is None, this if statement will always evaluate as "True", even if self.do_conv is true. I think the second part of th statement should be "... or do_conv is False",  no?

Reply:
Sorry, I made this update while post-bedtime braindead. I'll have this fixed up tonight

Reply:
Thanks! I'm not sure I understand why you wrote the second part of the statement as  "(do_conv is not None and not do_conv)" instead of "do_conv is false", but it seems to me that it should accomplish the same, so I'll mark this as resolved.